addons/l10n_es/data/account_tax_data.xml

```diff
@@ -1330,6 +1342,7 @@
         <field name="refund_account_id" ref="l10n_es.account_common_4751"/>
         <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         <field name="amount" eval="-2"/>
+        <field name="sequence" eval="2"/>
```

Comment:
Why do you need this?

Reply:
If you read the REAGYP you will see that IPRF if applied after apply VAT.

Reply:
OK, got it.